<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transponderDefinitionDetailsWidget</class>
 <widget class="QWidget" name="transponderDefinitionDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Uplink Frequency:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uplinkFreqEdit"/>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="downlinkFreqEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="textLabel4">
     <property name="text">
      <string>Hz</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel5">
     <property name="text">
      <string>Uplink Offset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="textLabel8">
     <property name="text">
      <string>Hz</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="uplinkOffsetEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="downlinkOffsetEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="modeEdit">
     <item>
      <property name="text">
       <string>SSB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CW</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="textLabel9">
     <property name="text">
      <string>Mode:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QCheckBox" name="preampCheckBox">
     <property name="text">
      <string>PreAmp</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="textLabel3">
     <property name="text">
      <string>Hz</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="textLabel6">
     <property name="text">
      <string>Hz</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="textLabel7">
     <property name="text">
      <string>Downlink Offset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Downlink Frequency:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="reverseCheckBox">
     <property name="text">
      <string>Reverse Tracking</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
